Inc/PrimitiveBatch.h

```diff
@@ -29,117 +29,119 @@
 
 namespace DirectX
 {
-    namespace Internal
+    inline namespace DX12
     {
-        // Base class, not to be used directly: clients should access this via the derived PrimitiveBatch<T>.
-        class PrimitiveBatchBase
+        namespace Private
```

Comment:
Had to change this namespace to avoid warnings with DirectXCollision.inl when I tried to add this to a inline namespace.